rootfs/etc/profile.d/direnv.sh

```diff
@@ -1,5 +1,4 @@
-PROMPT_HOOKS+=("direnv_prompt")
+# Install `direnv` via PROMPT_COMMAND hook
+#
 
-function direnv_prompt() {
-	eval "$(direnv hook bash)"
-}
+eval "$(direnv hook bash)"
```

Comment:
heh, thought I saw it had to be part of the `PROMPT_COMMAND` (which is what our `PROMPT_HOOKS`) manages, but now when I look at the instructions I don't find that and see what you did as the correct way.

Reply:
It does need to be part of the `PROMPT_COMMAND`, that what the eval does, it install it in the `PROMPT_COMMAND` string.